Gemfile

```diff
@@ -9,6 +9,12 @@ gem 'rake'
 gem 'rack-conneg', '~> 0.1.5'
 gem 'rack-rest_api_versioning', '~> 0.0.2'
 
+gem 'google-api-client'
+
+#gem "foreman", :path => "/Users/pje/my_foreman_fork"
+#gem "GGB", :path => "/Users/dlhaines/dev/BITBUCKET/ggb"
+gem "GGB", :path => "/Users/dlhaines/dev/GITHUB/dlh-umich.edu/FORKS/google_groups_gem"
```

Comment:
The explicit path is there so that the updates to the gem don't have to be published in order for the service to pick them up.  The final version will have a gem picked up from the git repo.